pymc3/step_methods/hmc/quadpotential.py

```diff
@@ -158,6 +166,7 @@ def __init__(self, n, initial_mean, initial_diag=None, initial_weight=0,
         self._background_var = _WeightedVariance(self._n, dtype=self.dtype)
         self._n_samples = 0
         self.adaptation_window = adaptation_window
+        self.adaptation_window_multiplier = int(adaptation_window_multiplier)
```

Comment:
I think it is better to leave this as float and convert adaptation_window below into int after multiplication.